packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -58,16 +93,11 @@ export const accordionItemStyles: (
       padding: 0 calc((6 + (${designUnit} * 2 * ${density})) * 1px);
       text-align: left;
       height: calc(${heightNumber} * 1px);
-      color: ${neutralForegroundRest};
+      color: inherit;
```

Comment:
It looks like this is needed because `.button` is a `button` element, which has a user agent style sheet setting the `color` to either white or black using the `internal-light-dark` function.